### DISCUSSION TITLE
#Lab1 after install Xilinx Vitis, didn't find settings64.sh

### DISCUSSION THREAD

**Discussion by lkl110137918218** [bol-edu/HLS-SOC-Discussions#62] [Q&A] [answered]
教授、助教、各位同學好，

我按照助教所提供的安裝步驟pdf來進行Xilinx Vitis 的安裝，
- 除了在前置安裝設置時，when I enter `sudo apt install build-essential -y`, 得到了以下錯誤訊息(第一張截圖)，所以依照終端機建議先下指令 `sudo apt-get update`，再下指令 `sudo apt install build-essential -y`，才通過
- 接著安裝Xilinx Vitis，及相關套件 `sudo ./installLibs.sh`，但卻在`source /tools/Xilinx/Vitis/2022.1/settings64.sh`時找不到該檔案(第二張截圖)
- 第三張截圖是安裝的log檔，不確定是否要將/tools/Xilinx砍掉重裝，想請問還是有甚麼其他可能原因或建議的做法，謝謝
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/122779900/fd8a14ce-bbcd-4f50-9447-0b926f819463)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/122779900/5b3d4540-1a2f-4cfc-a160-d1b8f13f8d7c)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/122779900/c8ffe2b1-b45d-4da0-a2e4-b3dd9cb3d071)

**Comment by boledulab** [comment, marked as answer]
嗨 請對照這份手冊裡的流程再確認一下 有沒有設置不同的地方
建議可以VM都全部刪除後 再重新安裝設置
[Vitis_on_Ubuntu_VM.pdf](https://github.com/bol-edu/HLS-SOC-Discussions/files/12694093/Vitis_on_Ubuntu_VM.pdf)
